scss/inc/_forms.scss

```diff
@@ -410,11 +410,17 @@ label {
             left: -1em;
         }
     }
+
     input[type="radio"],
     input[type="checkbox"] {
-        visibility: hidden;
-        pointer-events: none;
+        /* based on "visually-hidden" mixin in LDS for accessibility goals */
         position: absolute;
+        width: 1px;
+        height: 1px;
+        overflow: hidden;
+        clip: rect(1px, 1px, 1px, 1px);
+        margin: 0;
+        padding: 0;
```

Comment:
It's a good idea. I would even propose defining it as a `visuallyHidden` mixin in `scss/inc/_functions.scss`, because we've seen how useful the pattern was in Solar.